### PR TITLE
Feature/statistical factsheets template

### LINKED ISSUE #347: StatisticalFactsheets: create route and template view

### PR DESCRIPTION
**Pull request by minotogna**
Resolves #347

### CODE REVIEW

File: webapp/app/statisticalFactsheets/CarbonGrowingStock/CarbonGrowingStock.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import './carbonGrowingStock.less'
+import React from 'react'
+
+const CarbonGrowingStock = () => {
+  return (
+    <div className="statistical-factsheets-carbon-growing-stock">
+      <h3 className="header">Carbon and Growing Stock (1900 - 2000)</h3>
```
Comment (sorja) [inline webapp/app/statisticalFactsheets/CarbonGrowingStock/CarbonGrowingStock.js:7]: Are these supposed to be `1900 -` or `1990 -`?

File: webapp/app/statisticalFactsheets/ForestArea/ForestArea.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import './forestArea.less'
+import React from 'react'
+
+const ForestArea = () => {
+  return (
+    <div className="statistical-factsheets-forest-area">
+      <h3 className="header">Forest Area (1900 - 2000)</h3>
```
Comment (sorja) [inline webapp/app/statisticalFactsheets/ForestArea/ForestArea.js:7]: Same as above

File: webapp/app/statisticalFactsheets/ForestAreaWithinProtectedAreas/ForestAreaWithinProtectedAreas.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import './forestAreaWithinProtectedAreas.less'
+import React from 'react'
+
+const ForestAreaWithinProtectedAreas = () => {
+  return (
+    <div className="statistical-factsheets-forest-area-protected-areas">
+      <h3 className="header">Forest Area within Protected Areas (1900 - 2000)</h3>
```
Comment (sorja) [inline webapp/app/statisticalFactsheets/ForestAreaWithinProtectedAreas/ForestAreaWithinProtectedAreas.js:7]: Same as above

File: webapp/app/statisticalFactsheets/CarbonGrowingStock/CarbonGrowingStock.js (hunk 1 above)
Comment (minotogna) [reply webapp/app/statisticalFactsheets/CarbonGrowingStock/CarbonGrowingStock.js:7]: that's just a template. they will need to use i18n when implemented

File: webapp/app/statisticalFactsheets/ForestArea/ForestArea.js (hunk 2 above)
Comment (minotogna) [reply webapp/app/statisticalFactsheets/ForestArea/ForestArea.js:7]: same as above

File: webapp/app/statisticalFactsheets/ForestAreaWithinProtectedAreas/ForestAreaWithinProtectedAreas.js (hunk 3 above)
Comment (minotogna) [reply webapp/app/statisticalFactsheets/ForestAreaWithinProtectedAreas/ForestAreaWithinProtectedAreas.js:7]: same as above

**Pull request merged**